httphandler/roundtripper_decorators_test.go

```diff
@@ -100,7 +100,9 @@ func TestAccessLogging(t *testing.T) {
 	sendReq(t, srv, "GET", nil, rt)
 
 	amdstr := strings.Trim(buf.String(), "\n")
```

Comment:
You can use `Bytes()` and `bytes.Trim()` functions here, so that subsequent conversion to byte array will not be needed.